common/utils/EncryptionManager.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const crypto = require('crypto');
+const config = require('../config');
+const CONST = require('../constants');
+
+class EncryptionManager {
+
+  /**
+   * @param text test to be encrypted
+   */
+  encrypt(text) {
+    const cipher = crypto.createCipher(CONST.APISERVER.ENCRYPTION.AES_256_ALGORITHM, config.apiserver.encryption.password);
```

Comment:
Just a note: the createCipher function without an initialization vector is currently deprecated [see: https://nodejs.org/api/crypto.html#crypto_crypto_createcipher_algorithm_password_options ].

Reply:
Changed to createCipheriv